README.md

```diff
@@ -180,6 +180,19 @@ It is important that the remote instance is correctly pre-configured, with the P
 
 A base path for the frontend application can be established on the remote server by setting the corresponding field in the workflow inputs. This mechanism prevents conflicts between the frontend application and any pre-existing deployed application running on Payara, which can potentially be a Dataverse backend. This way, only the routes with the base path included will redirect to the frontend application.
 
+#### Beta Testing Environment
+
+To make the SPA Frontend accesible and testable by people interested in the project, there is a remote beta testing environment that includes the latest changes developed both for the frontend application and the Dataverse backend application (develop branches).
+
+This environment follows the "all-in-one" solution described above, where both applications coexist on a Payara server.
+
+Environment updates are carried out automatically through GitHub actions, present both in this repository and in the Dataverse backend repository, which deploy the develop branches when any change is pushed to them.
+
+The environment is accessible through the following URLs:
+
+- SPA: http://ec2-3-210-184-82.compute-1.amazonaws.com/spa
+- JSF: http://ec2-3-210-184-82.compute-1.amazonaws.com
```

Comment:
Should we set up a DNS entry for this? Something under dataverse.org?

I just saw "If we set a DNS for the environment before merging this PR, the README URLs will need to be updated."

Yes, I think we should do this, but what do we call it? I'm asking in Slack: https://iqss.slack.com/archives/C03R1E7T4KA/p1690813333254749

Reply:
We're going with http://beta.dataverse.org but we should copy the wildcard cert from https://dev1.dataverse.org over to it. @GPortas or @donsizemore I'm happy to work with you on this.